File: src/main/java/edu/marmara/model/Advisor.java
package edu.marmara.model;

import java.util.List;

public class Advisor extends Instructor {
    private List<Student> students;

    public Advisor() {
    }

    public Advisor(List<Student> students) {
        this.students = students;
    }

    public void addStudent(Student student){
        this.students.add(student);
    }

    @Override
    public String toString() {
        return "Advisor{" +
                "students=" + students +
                '}';
    }

    public List<Student> getStudents() {
        return students;
    }

    public void setStudents(List<Student> students) {
        this.students = students;
    }
}

File: src/main/marmara/model/Advisor.py
from src.main.marmara.model import Instructor

class Advisor(Instructor):
    def __init__(self, students=None):
        if students is None:
            students = []
        self.students = students

    def add_student(self, student):
        self.students.append(student)

    def __str__(self):
        return "Advisor(students={})".format(self.students)

    @property
    def students(self):
        return self._students

    @students.setter
    def students(self, students):
        self._students = students
